flink-python/pyflink/table/table_environment.py

```diff
@@ -1784,6 +1784,29 @@ def to_data_stream(self, table: Table) -> DataStream:
         """
         return DataStream(self._j_tenv.toDataStream(table._j_table))
 
+    def to_data_stream(self, table: Table, type_info: TypeInformation) -> DataStream:
+        """
+        Converts the given Table into a DataStream.
+
+        Since the DataStream API does not support changelog processing natively, this method
+        assumes append-only/insert-only semantics during the table-to-stream conversion. The records
```

Comment:
nit: the documentation only refers to insert-only. Can we remove append-only or is this indicating another semantic?